style.css

```diff
@@ -1,5 +1,31 @@
 @import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&display=swap");
 
+.menu-p {
+  width: 375px;
+  height: 755px;
+  position: fixed;
+  z-index: 2000;
+  left: -100%;
+  top: 0;
+  bottom: 57px;
+  background: #3c3a39;
+  mix-blend-mode: normal;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
+
+.menu-p.active {
+  width: 375px;
+  height: 755px;
+  position: fixed;
+  z-index: 2000;
+  left: 0;
+  top: 0;
+  bottom: 57px;
+  background: #3c3a39;
+  mix-blend-mode: normal;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
```

Comment:
Nice work implementing the mobile menu😉 But I think we can make it look even more like the design by giving the height property `height:100vh`  as it is a core requrement that we stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=1BajKMlmZn1jT0dS-0)

design menu
![fullwidth](https://user-images.githubusercontent.com/64643995/215542381-b5558918-3bbc-43c8-8efe-da2b98a9a816.png)

your menu
![image](https://user-images.githubusercontent.com/64643995/215542710-f7a65686-7796-4927-b4a8-651cd4ae589a.png)

kindly make the necessary changes😉